docs/includes/usage-examples/InsertManyTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use MongoDB\Laravel\Tests\TestCase;
+
+class InsertManyTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testInsertMany(): void
+    {
+        require_once __DIR__ . '/Movie.php';
+
+        Movie::truncate();
+
+        // begin-insert-many
+        $result = Movie::insert([
+            [
+                'title' => 'Anatomy of a Fall',
+                'year' => 2023,
```

Comment:
You should add an example with a DateTime object, because `casts` are not applied when `Model::insert` is used. I think we've already discussed this with @ccho-mongodb in another PR, which I can't find.

```php
'released_at' => new UTCDateTime(new DateTimeImmutable('2023-01-01'))
```

Reply:
Could you mention why we need to explicitly set the UTCDateTime? (because model casts are not applied).